src/renderer/renderer.ts

```diff
@@ -901,10 +902,36 @@ export class SDFGRenderer extends EventEmitter {
         else
             this.bgcolor = window.getComputedStyle(this.canvas).backgroundColor;
 
-        // Create the initial SDFG layout
-        // Loading animation already started in the file_read_complete function
-        // in sdfv.ts to also include the JSON parsing step.
+
         this.updateCFGList();
+
+        // Create the initial SDFG layout
```

Comment:
Since we're leaving this out for now, please remove this commented out section for this PR